NitroxClient/MonoBehaviours/Gui/MainMenu/JoinServer.cs

```diff
@@ -153,11 +153,11 @@ private void StartMultiplayerClient()
 
             try
             {
-                multiplayerSession.Connect(ServerIp);
+                multiplayerSession.Connect(ServerIp,serverPort);
             }
             catch (ClientConnectionFailedException)
             {
-                Log.InGame($"Unable to contact the remote server at: {ServerIp}:11000");
+                Log.InGame($"Unable to contact the remote server at: {ServerIp}:{serverPort.ToString()}");
```

Comment:
`ToString()` is not needed when you use interpolated strings (or string.Format).